bedrock/firefox/templates/firefox/whatsnew/includes/fx96-en/s2d.html

```diff
@@ -1,22 +0,0 @@
-{#
- This Source Code Form is subject to the terms of the Mozilla Public
- License, v. 2.0. If a copy of the MPL was not distributed with this
- file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#}
-
-{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
-
-<div class="mzp-c-emphasis-box">
-  <h3>Get Firefox for mobile</h3>
-{% if show_send_to_device %}
-  {{ send_to_device(include_title=False, input_label='Email yourself a download link', message_set='fx-whatsnew-96', spinner_color='#ccc;', class='vertical', id='send-to-device') }}
```

Comment:
Suggestion: can we remove the reference to the message set here? https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L1005